layers/+lang/python/packages.el

```diff
@@ -17,7 +17,6 @@
     company-anaconda
     cython-mode
     eldoc
-    ;; evil-matchit
```

Comment:
Maybe I don't understand how this works, but shouldn't be this line uncommented instead of deleted?